src/components/Table/DataTable/compenents/el-data-table/el-data-table.vue

```diff
@@ -1108,9 +1108,7 @@ export default {
     },
     handleSelectAll(e, canSelect) {
       this.loading = true
-      this.selectStrategy.onSelectAll(e, canSelect, () => {
-        this.loading = false
-      })
+      this.selectStrategy.onSelectAll(e, canSelect)
     },
     /**
      * 切换某一行的选中状态，如果使用了第二个参数，则是设置这一行选中与否
```

Comment:
There isn't much to check on here since it seems like a straightforward TypeScript import statement and event handler function definition with no apparent errors.

However, if anything is unclear to you:

  * The `export` keyword after a class declaration suggests that this file contains exported classes or other objects from which another module could import. Please note the scope of those definitions, particularly when dealing with interfaces/abstract types and properties.
   
  * Make sure that all imports should be valid in order not to break any built-in modules like `lodash` in TypeScript or ES6 features such as arrow functions.

  * Consider reviewing how you handle asynchronous operations within these methods (asynchronous loading states etc.), especially if they're used outside regular programming practices. Always ensure proper handling and management of async calls to prevent unnecessary delays or timeouts.